syntax = "proto3";

import "google/protobuf/timestamp.proto";

package intuit;

option java_package = "com.shahramatrian.intuit";
option java_outer_classname = "PersonProtos";
option java_multiple_files = true;

service PersonLogger {
    rpc LogPersonalInfo (PersonInfoRequest) returns (Response);
}

message PersonInfoRequest {
    string first_name = 1;
    string last_name = 2;
    string dob = 3;
    string email = 4;
    string phone_number = 5;
    google.protobuf.Timestamp timestamp = 6;
//    string timestamp = 6;
}

message Response {
    string message = 1;
}
